it/v2/endpoints/IgnoreEmploymentControllerHipISpec.scala

```diff
@@ -0,0 +1,134 @@
+      MtdIdLookupStub.ninoFound(nino)
+      setupStubs()
+      buildRequest(s"/$nino/$taxYear/$employmentId/ignore")
+        .withHttpHeaders(
+          (ACCEPT, "application/vnd.hmrc.2.0+json"),
+          (AUTHORIZATION, "Bearer 123")
+        )
+    }
+
+  }
+
+  "Calling the 'ignore employment' endpoint" should {
+    "return a 200 status code" when {
+      "any valid request is made" in new Test {
+
+        override def setupStubs(): Unit =
+          DownstreamStub.onSuccess(DownstreamStub.PUT, downstreamUri, downstreamQueryParams, status = NO_CONTENT, JsObject.empty)
+
+        val response: WSResponse = await(request.post(JsObject.empty))
+        response.status shouldBe OK
+        response.header("Content-Type") shouldBe None
+      }
+    }
+
+    "return error according to spec" when {
+      "validation error" when {
+        def validationErrorTest(requestNino: String,
+                                requestTaxYear: String,
+                                requestEmploymentId: String,
+                                expectedStatus: Int,
+                                expectedBody: MtdError): Unit = {
+          s"validation fails with ${expectedBody.code} error" in new Test {
+
+            override val nino: String         = requestNino
+            override val taxYear: String      = requestTaxYear
+            override val employmentId: String = requestEmploymentId
+
+            val response: WSResponse = await(request.post(JsObject.empty))
+            response.status shouldBe expectedStatus
+            response.json shouldBe Json.toJson(expectedBody)
+          }
+        }
+
+        val input = Seq(
+          ("AA1123A", "2019-20", "4557ecb5-fd32-48cc-81f5-e6acd1099f3c", BAD_REQUEST, NinoFormatError),
+          ("AA123456A", "20199", "78d9f015-a8b4-47a8-8bbc-c253a1e8057e", BAD_REQUEST, TaxYearFormatError),
+          ("AA123456A", "2019-20", "ABCDE12345FG", BAD_REQUEST, EmploymentIdFormatError),
+          ("AA123456A", "2018-19", "78d9f015-a8b4-47a8-8bbc-c253a1e8057e", BAD_REQUEST, RuleTaxYearNotSupportedError),
+          ("AA123456A", "2019-21", "4557ecb5-fd32-48cc-81f5-e6acd1099f3c", BAD_REQUEST, RuleTaxYearRangeInvalidError)
+        )
+        input.foreach(args => (validationErrorTest _).tupled(args))
+      }
+
+      "service error" when {
+        def serviceErrorTest(downstreamStatus: Int, downstreamCode: String, expectedStatus: Int, expectedBody: MtdError): Unit = {
+          s"downstream returns an $downstreamCode error and status $downstreamStatus" in new Test {
+
+            override def setupStubs(): Unit =
+              DownstreamStub.onError(DownstreamStub.PUT, downstreamUri, downstreamStatus, errorBody(downstreamCode))
+
+            val response: WSResponse = await(request.post(JsObject.empty))
+            response.status shouldBe expectedStatus
+            response.json shouldBe Json.toJson(expectedBody)
+          }
+        }
+
+        def errorBody(code: String): String =
```

Comment:
Update the error body to match the expected error response for this endpoint. Do the same for v1 and v2:
`s"""
             |[
             |   {
             |      "errorCode": "$code",
             |      "errorDescription": "message"
             |   {
             |]
            """.stripMargin`